angular/core/services/ability_service.coffee

```diff
@@ -148,7 +148,12 @@ angular.module('loomioApp').factory 'AbilityService', (AppConfig, Session) ->
       Session.user().locale != model.author().locale
 
     canEditPoll: (poll) ->
-      poll.isActive() and
+      poll.isActive() and @isPollAuthorOrGroupAdmin(poll)
+
+    canSetPollOutcome: (poll) ->
+      poll.isClosed() and @isPollAuthorOrGroupAdmin(poll)
```

Comment:
I'd like to stay away from group language within permissions (we're going to have to take it out for communities)

Can we call this 'canAdministerPoll'?